### DISCUSSION TITLE
Is there a way to define a STATE pin for Bluetooth Plugin?

### DISCUSSION THREAD

**Discussion by ProgressInIdeas** [grblHAL/plugins#27] [Q&A] [answered]
Is there a way to define a static pre defined STATE pin for the Bluetooth Plugin?  I have a defined pin set on my custom PCB, but don't know how to get the compiler to recognize this.  I'm using the Teensy 4.1 driver, and the standard Bluetooth HC-05 Plugin.

**Comment by terjeio** [comment, marked as answer]
Currently not (in code), which aux port to use can be configured with $385.
